lint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "inside $0"
+
+RET=0
+
+flake8 statsmodels --select=E901,E999,F822,F823,E111,E114
```

Comment:
Probably should add a comment: # TODO: Add F821 back in once all undefined names are resolved.

They are important as they each have the potential to raise NameError at runtime.

Reply:
Could you move this to a `setup.cfg`, so that editors pick up the settings as well?

```
[flake8]
exclude=build,docs,examples
select=E901,E999,...
```

Then you should just be able to have `flake8` here.